src/views/Graph.vue

```diff
@@ -526,6 +564,27 @@ export default {
           ]
         `)
       }
+
+      if (this.groupCycle) {
+        // Loop over the subgraphs
+        Object.keys(cycles).forEach((key, i) => {
+          // Loop over the nodes that are included in the subraph
+          const nodeFormattedArray = cycles[key].map(a => `"${a.id}"`)
+          ret.push(`
+          subgraph cluster_margin_${i}
+          {
+            margin=100.0
+            label="margin"
+            subgraph cluster_${i} {${nodeFormattedArray};\n
+              label = "${key}";\n
+              fontsize = 40
```

Comment:
This font size should match the one in the subgraph component (watch out for units, they may differ).